react/DeleteIcon/DeleteIcon.svg

```diff
@@ -1,18 +1 @@
-<?xml version="1.0" encoding="utf-8"?>
-<!-- Generator: Adobe Illustrator 16.0.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
-<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
-<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
-	 width="18px" height="21.009px" viewBox="0 0 18 21.009" enable-background="new 0 0 18 21.009" xml:space="preserve">
-<g>
-	<path d="M17.25,2.965h-3.84V0.749C13.41,0.336,13.074,0,12.66,0H5.223c-0.414,0-0.75,0.336-0.75,0.749v2.216H0.75
-		C0.336,2.965,0,3.301,0,3.715s0.336,0.749,0.75,0.749h0.694V20.26c0,0.413,0.336,0.749,0.75,0.749h13.521
-		c0.414,0,0.749-0.336,0.749-0.749V4.464h0.785c0.414,0,0.75-0.335,0.75-0.749S17.664,2.965,17.25,2.965z M5.972,1.499h5.939v1.466
-		H5.972V1.499z M14.966,19.51H2.943V4.487h2.279h7.438h2.306V19.51z"/>
-	<path d="M5.962,5.254c-0.414,0-0.749,0.336-0.749,0.75V17.99c0,0.414,0.335,0.75,0.749,0.75s0.75-0.336,0.75-0.75V6.004
-		C6.712,5.59,6.376,5.254,5.962,5.254z"/>
-	<path d="M8.965,5.254c-0.414,0-0.75,0.336-0.75,0.75V17.99c0,0.414,0.336,0.75,0.75,0.75s0.75-0.336,0.75-0.75V6.004
-		C9.715,5.59,9.379,5.254,8.965,5.254z"/>
-	<path d="M11.963,5.254c-0.414,0-0.749,0.336-0.749,0.75V17.99c0,0.414,0.335,0.75,0.749,0.75s0.75-0.336,0.75-0.75V6.004
-		C12.713,5.59,12.377,5.254,11.963,5.254z"/>
-</g>
-</svg>
+<svg xmlns="http://www.w3.org/2000/svg" width="18" height="21.009"><path d="M17.25 2.965h-3.84V.749A.75.75 0 0 0 12.66 0H5.223a.75.75 0 0 0-.75.749v2.216H.75a.75.75 0 0 0 0 1.499h.694V20.26a.75.75 0 0 0 .75.749h13.521a.75.75 0 0 0 .749-.749V4.464h.785a.75.75 0 1 0 .001-1.499zM5.972 1.499h5.939v1.466H5.972V1.499zm8.994 18.011H2.943V4.487h12.023V19.51z"/><path d="M5.962 5.254a.75.75 0 0 0-.749.75V17.99a.75.75 0 1 0 1.499 0V6.004a.75.75 0 0 0-.75-.75zM8.965 5.254a.75.75 0 0 0-.75.75V17.99a.75.75 0 0 0 1.5 0V6.004a.75.75 0 0 0-.75-.75zM11.963 5.254a.75.75 0 0 0-.749.75V17.99a.75.75 0 1 0 1.499 0V6.004a.75.75 0 0 0-.75-.75z"/></svg>
```

Comment:
Has anything changed in this SVG? Is this just changes to whitespace?

Reply:
🤔... not sure if we should version svgs like this given a lot of styleguide consumers would be using svgo as part of their webpack config. Any other opinions on this?

Reply:
@georgespyropoulos RE: Breaking change. The result of this change is as follows:

- Plus icon: 20px x 20px (before) vs. 16px x 16px (after)
- DeleteIcon: 18px x 21px (before) vs. 16px x 16px (after)

So would if be fair to say that it's only a breaking change if consumers have applied CSS rules to adjust the size as this update might caused visual differences?

Happy to be advised on this one as I'm unfamiliar with what constitutes a breaking change in this instance

Reply:
Icons are styles with 16px by default with the Icon component, so there will be no difference for consumers @georgespyropoulos. We should've them compressed I guess from the start I guess

Reply:
Interestingly changes the icon 🤔 Maybe will agree to revert the compression for now

Reply:
You might need to hand edit the svgo output a little.  We've converted all the icons in `seek-asia-style-guide` but you probably want to go to `viewBox`, see if there's any weird fill rules or masks or whatever that came through.

I don't *really* expect to see that on style guide icons rather than random icons pulled in from specs, but that's the process I've been using this week to convert unoptimised source icons.

Reply:
This has been reverted along with the compression change to `PlusIcon.svg`